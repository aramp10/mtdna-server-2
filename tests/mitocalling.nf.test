    nextflow_pipeline {

    name "Test Workflow main.nf"
    script "main.nf"
    
    test("Runs with single BAM file") {

        when {
            params {
               project = "test-job"
               files   = "$projectDir/tests/data/bam/*.bam"
               output  = "out"
            }
        }

        then {
            assert workflow.success
            assert snapshot(new File("out"), workflow.trace).match()
        }

    }

    }