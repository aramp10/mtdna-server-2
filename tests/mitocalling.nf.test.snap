{
    "Runs with BAM file including different header contigs": {
        "content": [
            "variants_ann.txt:md5,c2aa6c45193a921f4a20238fe9bf6cd8"
        ],
        "timestamp": "2024-01-10T13:45:32.689184313"
    },
    "Runs with single BAM file and mutserve": {
        "content": [
            "variants_ann.txt:md5,606016ca87ca11897119066253d53c62"
        ],
        "timestamp": "2024-01-10T13:47:14.24639646"
    }
}